content/tutorials/history.textile

```diff
@@ -160,7 +226,7 @@ blang[javascript].
     channel.publish("play", "cluck");
   ```
 
-  "See this step in Github":https://github.com/ably/tutorials/commit/f460ba7
+  "See this step in Github":https://github.com/ably/tutorials/commit/f460ba7 
```

Comment:
Extra space, you should have your editor configured to trim white space.  Also, is this SHA valid?